cmake/CMakeLists.txt

```diff
@@ -0,0 +1,178 @@
+cmake_minimum_required(VERSION 3.9)
+enable_language(Fortran)
+
+# Load lists of source files
+include(Files.cmake)
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES "GNU")
+    set(dialect "-fdefault-real-8 -fdefault-double-8 -ffree-line-length-none -fbacktrace")
+    set(bounds "-fbounds-check")
+    if(NOT CMAKE_Fortran_COMPILER_VERSION VERSION_GREATER_EQUAL 8.0.0)
+      set(DISABLE_OMP ON)
+    endif()
+endif()
+if(CMAKE_Fortran_COMPILER_ID MATCHES "Intel")
+    set(dialect "-axAVX2 -r8 -traceback")
+    set(bounds "-check bounds")
+endif()
+set(CMAKE_Fortran_FLAGS_DEBUG "${CMAKE_Fortran_FLAGS_DEBUG} ${bounds}")
+set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} ${dialect}")
+
+# Populate xtb_version.fh
+set(version ${PROJECT_VERSION})
+execute_process(COMMAND git rev-parse HEAD
+ RESULT_VARIABLE git_return
+ OUTPUT_VARIABLE commit)
+string(REGEX REPLACE "\n$" "" commit ${commit})
+string(TIMESTAMP date "%m/%d/%Y")
+set(author $ENV{USERNAME})
+set(origin $ENV{HOSTNAME})
```

Comment:
`HOSTNAME` is not always set, for example on my machine and the cluster in Bonn it is always empty.

Reply:
I will switch it to a CMake variable.